FrontEnd/.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+package-lock.json
+package.json
```

Comment:
I double it should be included into `.gitignore`.

Reply:
But why? How other devs will receive updated packages? `package-lock.json` and `package.json` should not be ignored by git